spec/controllers/hyrax/my/collections_controller_spec.rb

```diff
@@ -1,29 +1,22 @@
 describe Hyrax::My::CollectionsController, type: :controller do
   describe "logged in user" do
-    let(:user)  { create(:user) }
-    let(:other) { create(:user) }
-
-    let!(:my_file)              { create(:work, user: user) }
-    let!(:first_collection)     { create(:public_collection, user: user) }
-    let!(:unrelated_collection) { create(:public_collection, user: other) }
-
-    before { sign_in user }
-
     describe "#index" do
-      context "with mulitple pages of collections" do
-        before { 2.times { create(:public_collection, user: user) } }
-        it "paginates" do
-          get :index, params: { per_page: 2 }
-          expect(assigns[:document_list].length).to eq 2
-          get :index, params: { per_page: 2, page: 2 }
-          expect(assigns[:document_list].length).to be >= 1
-        end
-      end
+      let(:user)  { create(:user) }
+      let(:other) { create(:user) }
+      before { sign_in user }
+
+      let!(:my_file)              { create(:work, user: user) }
```

Comment:
If you *really* want to make this fast you could just test that the correct search_builder is used. Alternately, just add fixtures to solr (not to fedora) as this controller action should not be hitting fedora.